### PR TITLE
build: configure base modules

### PR DESCRIPTION
**Pull request by shoota**
## TL;DR

- [x] add general modules to root package.json as dev dependencies.
- [x] configure commit hook
- [x] edit README
- [x] setup circle ci project

## check list
- [x] pass ci
  - yarn test
  - yarn lint
  - ~yarn tsc~

### CODE REVIEW

File: packages/eslint-config-globis/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "react",
+    "es",
+    "globis",
+    "jsx",
+    "typescript"
+  ],
+  "scripts": {
+    "test": "echo test",
+    "tsc": "echo tsc",
+    "lint": "echo lint"
```
Comment (shoota) [inline packages/eslint-config-globis/package.json:22]: dummy scripts. 
After merge this PR and implements detail, should fix these 😉

File: packages/eslint-config-globis/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "lint",
+    "react",
+    "es",
+    "globis",
+    "jsx",
+    "typescript"
+  ],
+  "scripts": {
+    "test": "echo test",
+    "tsc": "echo tsc",
```
Comment (dkimura) [inline packages/eslint-config-globis/package.json]: why `tsc` is necessary? needed for eslint-config-globis?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      - image: circleci/node:lts-browsers
+    working_directory: ~/repo
+
+commands:
+  install_node_modules:
+    description: 'install npm dependencies'
+    steps:
+      - restore_cache:
+          keys:
+            - v1.1-dependencies-{{ checksum "package.json" }}
```
Comment (dkimura) [inline .circleci/config.yml]: badge
`v1` ?

**Review by dkimura — APPROVED** [review]

**Pull request merged**